src/pages/Signup.vue

```diff
@@ -45,14 +45,11 @@
         </checkbox-with-validation>
```

Comment:
@comteharbour why do we have a `Signup` component in the `consumer-frontend` code?